test/functional/admin/promotional_features_controller_test.rb

```diff
@@ -80,7 +80,6 @@ class Admin::PromotionalFeaturesControllerTest < ActionController::TestCase
 
     delete :destroy, params: { organisation_id: @organisation, id: promotional_feature }
 
-    assert_redirected_to admin_organisation_promotional_features_url(@organisation)
```

Comment:
Just wondering why this was removed? I'd expect this to still pass with the new implementation